### ISSUE TITLE
Add `stringify` capability

### ISSUE THREAD

**Issue by blai** [jsonmaur/zoo#1]
See https://github.com/dominictarr/bash-vars#stringify

**Comment by jsonmaur** [comment]
Interesting. So when would something like this be useful? Seems like you'd want to do it the other way around, writing your environment variables in `.env` in "stringified mode", then using them wherever you need them, whether in Node or Bash. Prepending any cli command with `zoo` will set the variables.

**Comment by blai** [comment]
In practical application development, assume we have a `.env.default` (which is modified whenever new variables are introduced for the app), `.env.default` get checked in as part of the source code, but it isn't used (or merely used as an example) in the app runtime. And everyone have a local `.env` (that is part of `.gitignore`), we need a way to keep the two in sync, which we decided to have a script that run on `post-checkout/rebase` so everyone gets the default of the new env var. This involve merging two env files, so that is a good use case there. The code itself is quite small for this part, so we didn't miss much if it isn't part of this module.

```js
function stringifyConfig(config) {
  return Object.keys(config)
    .map((key) => [key, config[key]].join('='))
    .join('\n')
  ;
}
```

**Comment by jsonmaur** [comment]
Sure, makes sense. I'll add this method for you and release sometime today :)

**Comment by jsonmaur** [comment]
This is now available in `0.1.9`. Thanks for the request @blai!
